tools/operator/testclient/operations/executeCommand.go

```diff
@@ -0,0 +1,21 @@
+package operations
+
+import (
+	"github.com/hyperledger/fabric-test/tools/operator/networkclient"
+	"github.com/hyperledger/fabric-test/tools/operator/testclient/inputStructs"
+)
+
+//DoCommandAction --
+func DoCommandAction(config inputStructs.Config) error {
+
+	var err error
+	for index := 0; index < len(config.CommandOptions); index++ {
+		args := config.CommandOptions[index].Args
+		name := config.CommandOptions[index].Name
+		_, err = networkclient.ExecuteCommand(name, args, true)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
```

Comment:
Doing it this way, we can only ever run  a single series of commands, maybe we can make `action` itself and object that has two properties `command` and `subcommand` (we can debate naming) and we change the name of `command` in the yaml to `script` or `raw`. Then subcommand is a `map[string]string[]` and if `command == script` we index into the subcommand map and pull out the args and run only that one raw command.

Reply:
This will allow us to specify as many raw commands as we want and call them when we need to